MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include REQUIREMENTS.txt
 recursive-include sptx_format/schema/ *.json
 recursive-include sptx_format/examples/ *.json
+recursive-include notebooks/data/ *.pkl
```

Comment:
Is it scalable that we always check in the results?  Also, does it need to be included in the pip pkg?  Looks like you're using a relative path to reference them anyway...